src/components/Page/components/Header/Header.tsx

```diff
@@ -137,18 +138,22 @@ class Header extends React.PureComponent<CombinedProps, State> {
         <div className={styles.Navigation}>
           {breadcrumbMarkup}
           {paginationMarkup}
+          {breadcrumbMarkup && rollupMarkup}
         </div>
       ) : null;
 
     const titleMarkup = (
-      <div className={styles.Title}>
-        {/* Anonymous divs are here for layout purposes */}
-        <div>
-          <DisplayText size="large" element="h1">
-            {title}
-          </DisplayText>
+      <div className={styles.TitleAndRollup}>
+        <div className={styles.Title}>
+          {/* Anonymous divs are here for layout purposes */}
```

Comment:
Just a side note, no need to do anything here, but when I see a comment like this I wonder if we can instead write the code in a more meaningful way like creating a component called `AnonymousDivForLayout` that just renders a div.